lib/index.js

```diff
@@ -104,6 +105,13 @@ class PackageJson {
     return p.fix(opts)
   }
 
+  // fix only the name field (for publishing)
+  static async fixName (path) {
```

Comment:
we don't need a top level export for this we'll just have libnpmpublish add this to its normalize steps.